### ISSUE TITLE
Doesn't show the errors

### ISSUE THREAD

**Issue by paranoicsan** [AtomLinter/linter-shellcheck#113]
What I've got
- macOS
- `shellcheck` is installed via homebrew and is inside $PATH
```shell
$ shellcheck -V
ShellCheck - shell script analysis tool
version: 0.4.7
```
- manual check works as it should
```shell
$ shellcheck .git/hooks/pre-commit
In .git/hooks/pre-commit line 1:
#!/usr/bin/env bash -l
^-- SC2096: On most OS, shebangs can only specify a single parameter.
```

But no error in the Atom. Developer console doesn't contain any errors either. Other linters work just fine.

Any help?

**Comment by Arcanemagus** [comment]
Can you run `Linter: Debug` from the Command Palette?

**Comment by paranoicsan** [comment]
Hm. It's weird. Debug info contains the line
```
Opened file is ignored: Yes
```

I copied the file outside the `.git` directory and I do see the linter error. Any thoughts where I can tweak the `opened file is ignored` setting?

**Comment by Arcanemagus** [comment]
You can either disable the core Atom setting of "Ignore VCS ignored files", which `linter` and many other packages follow, or you can change your `.gitignore` to not ignore the file 😉.

**Comment by paranoicsan** [comment]
Nice! Thanks! 👍
